README.md

```diff
@@ -0,0 +1,29 @@
+# How are the docs on openliberty.io built and what is a docs playbook?
+
+The docs on https://openliberty.io/ are built using a tool called Antora(https://antora.org/). It is a tool that helps streamline writing, managing, and versioning documentation.
+
+The docs playbook is a key file used by Antora see https://docs.antora.org/antora/latest/playbook/. It tells the build what branches and versions of the [docs repo](https://github.com/OpenLiberty/docs) to use. Antora aggregates all of the content specified in the playbook and creates versions of pages based on the `version` attribute in the `antora.yml` of each doc branch. See https://github.com/OpenLiberty/docs/blob/vNext/antora.yml for an example. If Antora finds the same doc in multiple doc branches, it will create a version picker on the site that lets you easily switch between versions of the doc.
+
+There are three doc playbooks used in the UI builds: prod, staging, and draft.
```

Comment:
"There are three doc playbooks used in the builds, located in the following branches in this repo:" and make each of these a link, so:
https://github.com/OpenLiberty/docs-playbook/blob/prod/antora-playbook.yml
https://github.com/OpenLiberty/docs-playbook/blob/staging/antora-playbook.yml
https://github.com/OpenLiberty/docs-playbook/blob/draft/antora-playbook.yml